### ISSUE TITLE
irazasyed/telegram-bot-sdk v3.3.0 requires php ^7.2 which does not match your installed version 8.0.3.

### ISSUE THREAD

**Issue by QueenZe** [irazasyed/telegram-bot-sdk#898]
Hello, 

I have an issue right now.
I can't install this package. i try to install v3.3.0 and this is the return value

```
 [InvalidArgumentException]
  Package irazasyed/telegram-bot-sdk at version 3.3.0 has a PHP requirement incompatible with your PHP version, PHP e
  xtensions and Composer version:
    - irazasyed/telegram-bot-sdk v3.3.0 requires php ^7.2 which does not match your installed version 8.0.3.
```

can someone help me to resolve this issue?

![gambar_2021-10-18_164245](https://user-images.githubusercontent.com/45986058/137707491-f5dc3ea8-c010-4b92-a334-22aa0b0cd8c5.png)

**Comment by edoc-tibens** [comment]
Please use / wait for v3.4.0 / v3.4.1

**Comment by Chouchen** [comment]
PHP 8 is available since v3.4.0 : https://packagist.org/packages/irazasyed/telegram-bot-sdk#v3.4.0

This should be closed now.
